agent/tool-scripts/lockstat

```diff
@@ -107,19 +107,28 @@ done
 
 tool_dir="$dir/tools-$group"
 tool_output_dir=$tool_dir/$tool # all tools keep data in their tool specific dir
-tool_ouput_file=$tool_output_dir/$tool.txt
+tool_output_file=$tool_output_dir/$tool.txt
 
 case "$mode" in
 	install)
-        ;;
+	if [[ ! -f "/proc/lock_stat" ]]; then
+		echo "please check if the kernel is built with CONFIG_LOCK_STAT=y in order to collect statistics on locks"
+        	error_log "[$script_name]: The kernel is not built with CONFIG_LOCK_STAT=y"
+		exit 1
+        fi
+	# Enable collection of statistics
+	echo 1 >/proc/sys/kernel/lock_stat
+	;;
 	start)
 	mkdir -p $tool_output_dir
-        echo "time: `date +%s.%N`" >>$tool_output_file
-	cat /proc/lock_stat >> $tool_output_file
-        ;;
-        stop)
-        echo "time: `date +%s.%N`" >>$tool_output_file
-	cat /proc/lock_stat >> $tool_output_file
+	echo "time: `date +%s.%N`" >> "$tool_output_file"
+	cat /proc/lock_stat >> "$tool_output_file"
+	;;
+	stop)
+	echo "time: `date +%s.%N`" >> "$tool_output_file"
+	cat /proc/lock_stat >> "$tool_output_file"
+	# Disable collection of statistics
+	echo 0 >/proc/sys/kernel/lock_stat
```

Comment:
If stats were enabled before, then maybe they should not be disabled afterwards.